scripts/eateries/EaterySelect.js

```diff
@@ -5,9 +5,13 @@ const eventHub = document.querySelector(".container")
 
 contentTarget.addEventListener("change", (changeEvent) => {
     if (changeEvent.target.id === "dropdown--eatery") {
-        
         const selectedEateryId = changeEvent.target.value
-        const selectedEateryName = useEateryById(selectedEateryId).businessName
+        const selectedEatery = useEateryById(selectedEateryId)
+
+        let selectedEateryName = ""
+        if(selectedEatery) {
+            selectedEateryName = selectedEatery.name
```

Comment:
I noticed that the key we are using here changed from `buisnessName` to `name`. It still renders to the DOM, but is that the key we want to be using?